packages/components/src/tabs/tests/chromatic/Tabs.chroma.jsx

```diff
@@ -0,0 +1,686 @@
+import { PlaceholderMajorIcon, CrossMajorIcon, NotificationMajorIcon } from "@components/icons";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong